xwiki-platform-core/xwiki-platform-skin/xwiki-platform-skin-ui/src/main/resources/SkinsCode/Translations.xml

```diff
@@ -47,32 +47,32 @@ skin.sheet.addTemplate=Add a new overriding template
 skin.sheet.add=Add
 skin.sheet.overrideTemplateWarning=This section allows you to override a template of the skin. This is technical and you should be aware of the consequences.
 skin.sheet.deprecatedPropertiesWarning=This section is here for retro-compatibility reasons. You should use the 'overridden templates' mechanism above.
-skin.sheet.nojs=Without JavaScript, you cannot use this section. But you can use the {0}Object Editor{1} instead, and manually add {2} objects.
+skin.sheet.nojs=Without JavaScript, you cannot use this section,but you can use the {0}Object Editor{1} instead, and manually add {2} objects.
```

Comment:
you should add a space after the comma

Reply:
Personally I prefer short sentences. If the issue is starting a sentence with "But" then an easy solution is to replace it with "However". You might as well rewrite the sentence in this case... "You cannot use this section without Javascript enabled. However you could use the {0}Object Editor{1}, and manually add {2} objects."